minimal.slim

```diff
@@ -95,26 +103,36 @@ function (void)setupParams(object<Dictionary>$ defaults)
 	if (!exists("OUTDIR")) defineConstant("OUTDIR", ".");
 	defaults.addKeysAndValuesFrom(Dictionary("PARAMFILE", PARAMFILE, "OUTDIR", OUTDIR));
 	
-	if (fileExists(PARAMFILE)){
+	if (fileExists(PARAMFILE)) {
+		// BCH: Does this code work?  I'm trying to figure out what that sep="\n" is doing there and why
```

Comment:
It does not work,; this code is only triggered if `params.json` is present in the working directory (observe this by running `slim minimal.slim` in this directory). I've fixed this in #19 .